docs/iceberg-rest-service.md

```diff
@@ -55,7 +63,7 @@ You must set `gravitino.auxService.iceberg-rest.httpPort` explicitly, like `9001
 ### Iceberg catalog configuration
 
 :::info
-The Gravitino Iceberg REST catalog service using memory catalog for default. You can specify Hive or JDBC catalog for production environments.
+The Gravitino Iceberg REST catalog service using memory catalog for default. You can specify Hive or JDBC catalog in production environments.
```

Comment:
The Gravitino Iceberg REST catalog service uses the memory catalog by default. You can specify using a Hive or JDBC catalog in a production environment.

OR 

The Gravitino Iceberg REST catalog service uses the memory catalog by default. You can specify using Hive or JDBC catalog for production environments.

Reply:
done